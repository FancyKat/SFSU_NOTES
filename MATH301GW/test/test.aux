\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand*\HyPL@Entry[1]{}
\bibstyle{biblatex}
\bibdata{test-blx,references}
\citation{biblatex-control}
\abx@aux@refcontext{nty/global//global/global}
\providecommand\babel@aux[2]{}
\@nameuse{bbl@beforestart}
\HyPL@Entry{0<</S/D>>}
\babel@aux{english}{}
\citation{Eg}
\abx@aux@cite{0}{Eg}
\abx@aux@segm{0}{0}{Eg}
\@writefile{toc}{\contentsline {section}{\numberline {1}Examples}{3}{section.1}\protected@file@percent }
\@writefile{loe}{\contentsline {theorem}{\ifthmt@listswap Theorem~1.1\else \numberline {1.1}Theorem\fi }{3}{theorem.1.1}\protected@file@percent }
\@writefile{loe}{\contentsline {proposition}{\ifthmt@listswap Proposition~1.2\else \numberline {1.2}Proposition\fi }{3}{proposition.1.2}\protected@file@percent }
\@writefile{loe}{\contentsline {principle}{\ifthmt@listswap Principle~1.3\else \numberline {1.3}Principle\fi }{3}{principle.1.3}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {1.1}Pictures}{3}{subsection.1.1}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {1.2}Citation}{3}{subsection.1.2}\protected@file@percent }
\abx@aux@read@bbl@mdfivesum{nohash}
\gdef \@abspage@last{4}
