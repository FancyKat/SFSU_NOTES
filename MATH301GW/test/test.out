\BOOKMARK [1][-]{section.1}{\376\377\000E\000x\000a\000m\000p\000l\000e\000s}{}% 1
\BOOKMARK [2][-]{subsection.1.1}{\376\377\000P\000i\000c\000t\000u\000r\000e\000s}{section.1}% 2
\BOOKMARK [2][-]{subsection.1.2}{\376\377\000C\000i\000t\000a\000t\000i\000o\000n}{section.1}% 3
